yb-voyager/cmd/live_migration.go

```diff
@@ -176,9 +193,9 @@ func processEvents(chanNo int, evChan chan *tgtdb.Event, done chan bool) {
 		if len(batch) == 0 {
 			continue
 		}
-		err := tdb.ExecuteBatch(batch)
+		err := tdb.ExecuteBatch(migrationUUID, tgtdb.EventBatch{Events: batch, ChanNo: chanNo})
 		if err != nil {
-			utils.ErrExit("error executing batch: %v", err)
+			utils.ErrExit("error executing batch: %w", err)
```

Comment:
Error message should include some identification of a "batch" e.g. tableName, fileName, batchNumber.

Reply:
There isn't really any identification of a batch, so for now, just included the channel No. The error will anyway include the statement that failed as well.